src/CacheInvalidator.php

```diff
@@ -198,6 +206,27 @@ public function invalidate(array $headers)
     }
 
     /**
+     * Remove/Expire cache objects based on cache tags
+     *
+     * @see TagsInterface::tags()
+     *
+     * @param array $tags Tags that should be removed/expired from the cache
+     * @param string $tagsHeader Name of the HTTP header used to provide the tags
+     *
+     * @throws UnsupportedProxyOperationException If HTTP cache does not support Tags invalidation
+     *
+     * @return $this
+     */
+    public function invalidateTags(array $tags, $tagsHeader = 'X-Cache-Tags')
```

Comment:
leaving $tagsHeader out of here would also have the benefit of keeping this method the same as we had in version 1 (and deprecated because the TagHandler was supposed to do this).